demo/admin/src/common/MasterMenu.tsx

```diff
@@ -89,6 +90,13 @@ export const masterMenuData: MasterMenuData = [
                     component: NewsPage,
                 },
             },
+            {
+                primary: <FormattedMessage id="menu.translator" defaultMessage="Translator" />,
+                route: {
+                    path: "/structured-content/translator",
+                    component: TranslatorPage,
+                },
+            },
```

Comment:
I'd prefer if you'd demo the translation feature in the page tree, for instance `EditPage`